progs/symmetrc_mimc.py

```diff
@@ -0,0 +1,69 @@
+from random import randint
+from asyncio import gather
+from honeybadgermpc.field import GF
+from honeybadgermpc.mpc import PreProcessedElements, Subgroup
+from honeybadgermpc.elliptic_curve import Point, Jubjub
+from progs.jubjub import share_mul
+from progs.mimc import mimc_mpc, mimc_plain
+
+pp_elements = PreProcessedElements()
+field = GF(Subgroup.BLS12_381)
+CURVE = Jubjub()
+KEY_LENGTH = 32
+# GP: The generator of Jubjub curve, hardcode here
+GP = Point(5,
+           6846412461894745224441235558443359243034138132682534265960483512729196124138,
+           CURVE)
+
+
+async def key_generation(context):
+    """
+    The MPC system creates random bitwise shared value [x]_B
+    as the private key (priv_key),
+    then calcultes X = ([x]G).open as the public key (pub_key)
+    """
+    # Generate the private key
+    priv_key = [pp_elements.get_bit(context) for _ in range(KEY_LENGTH)]
+    # priv_key_p = field(0)
```

Comment:
Make sure to clean up commented code, or provide a reason why it's commented out